source/documentation/guidance/buildpacks.md

```diff
@@ -1,44 +1,120 @@
+			</tr>
+			<tr>
+				<td style="vertical-align: middle; text-align: left">
+					Base operating system
+					<br> - update OS libraries
+				</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #ffbf47">Yours</td>
+			</tr>
+			<tr>
+				<td style="vertical-align: middle; text-align: left">
+					App admin
+					<br> - ensure app instances are up and running
+					<br> - ensure provisioning capacity
+          <br> - stream application logs
+				</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+			</tr>
+			<tr>
+				<td style="vertical-align: middle; text-align: left">
+					<a href="https://docs.cloud.service.gov.uk/#deploy-a-backing-or-routing-service">Backing services</a>
+					<br> - maintain service availability
+					<br> - apply security patches
+				</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+			</tr>
+			<tr>
+				<td style="vertical-align: middle; text-align: left">
+					Physical hardware
+					<br>- resolve hardware failures
+					<br>- encrypt network data traffic
+				</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+
+### Standard buildpacks
+
+You are only responsible for the app code and its dependencies. This includes [penetration testing](https://docs.cloud.service.gov.uk/#penetration-testing) your app.
 
-#### Standard buildpack
-You are only responsible for the app code and its dependencies.
+If you're relatively new to cloud or have a small development team, we recommend using the standard buildpacks to maximise the support you will receive from GOV.UK PaaS.
 
-#### Custom buildpacks
+### Custom buildpacks
 
-You are responsible for managing this custom buildpack as well as the app code and its dependencies. This applies both in setting up the app, and running it in production.
+You are responsible for managing this custom buildpack's language and runtime, as well as the app code and its dependencies. This applies both in setting up the app, and running it in production.
```

Comment:
We are not really talking about a specific custom buildpack, so use `for managing a custom buildpack's`. Also `This applies both _to_`

Reply:
The section is *Custom buildpacks*

I do not understand the other comment `This applies both _to_`, I guess you refer to grammar. @jonathanglassman any comment on this?

Reply:
My first comment is about replacing `this` with `a`.
The second one is to replace `in` with `to`.

Reply:
Agree with @bandesz. Will change to `You are responsible for managing a custom buildpack's language and runtime, as well as the app code and its dependencies. This applies both to setting up the app, and running it in production.`